biothings/utils/web/analytics.py

```diff
@@ -70,6 +86,8 @@ def ga_track(self, event={}):
                 request_body += '&ec={}&ea={}'.format(event['category'], event['action'])
                 if event.get('label', False) and event.get('value', False):
                     request_body += '&el={}&ev={}'.format(event['label'], event['value'])
+                if referrer:
+                    request_body += f'&dr={referrer}'
 
             req = HTTPRequest('http://www.google-analytics.com/batch', method='POST', body=request_body)
```

Comment:
Based on [this](https://developers.google.com/analytics/devguides/reporting/core/v3/batching), `http://www.google-analytics.com/batch` should have already deprecated. We should use their new API endpoint? Hope it's just URL change.

Reply:
Actually it's different, we are using `http://www.google-analytics.com/batch` not `www.googleapis.com/batch`